back/engines/free/email_campaigns/spec/mailers/footer_spec.rb

```diff
@@ -19,9 +19,10 @@
   end
 
   let(:mail) { EmailCampaigns::CommentOnIdeaYouFollowMailer.with(command: command, campaign: campaign).campaign_mail.deliver_now }
+  let(:body) { mail_body(mail) }
```

Comment:
This was necessary because the body would be split across multiple lines with special characters in-between, which was problematic for matching in the specs.